tests/stencils/test_sparse_fill.py

```diff
@@ -0,0 +1,75 @@
+from dusk.script import *
+from test_util import transpile_and_validate
+
+
+def test_sparse_fill():
+    transpile_and_validate(sparse_order_2_fill)
+    transpile_and_validate(longer_fills)
+    transpile_and_validate(fill_with_reduction)
+
+
+@stencil
+def sparse_order_2_fill(
+    vertex: Field[Vertex],
+    edge: Field[Edge, K],
+    cell: Field[Cell],
+    ve: Field[Vertex > Edge, K],
+    vc: Field[Vertex > Cell, K],
+    ev: Field[Edge > Vertex, K],
+    ec: Field[Edge > Cell, K],
+    cv: Field[Cell > Vertex, K],
+    ce: Field[Cell > Edge, K],
+):
+
+    with levels_upward:
+
+        with sparse[Vertex > Edge]:
+            ve = edge + 5
+        with sparse[Vertex > Cell]:
+            vc = cell - 6
+        with sparse[Edge > Vertex]:
+            ev = vertex * 7
+        with sparse[Edge > Cell]:
+            ec = cell / 8
+        with sparse[Cell > Vertex]:
+            cv = vertex ** 9
+        with sparse[Cell > Edge]:
+            ce = sqrt(edge)
```

Comment:
Suggestion: add dense accesses to rhs of fill

Reply:
I guess you meant sparse accesses.

Reply:
Sorry, I didn't specify which dense accesses I referred to (the starting location ones).
E.g.:
```
        with sparse[Vertex > Edge]:
            ve = vertex + edge + 5
```
You can keep the sparse accesses that you added for completeness, it doesn't harm :)

Reply:
Added them.